math_hurdler.py

```diff
@@ -220,6 +207,19 @@ def run(self):
         hurdle_y = display_info.current_h - \
             hurdle.get_height() - (2 * ground.get_height() / 3) + 70
 
+        self.buttons = []
+        button_width = 200
+        button_height = 80
+        buttons_x_padding = 50
+        buttons_y_padding = (ground.get_rect().height - grass.get_rect().height - button_height) / 4
+        buttons_gap = (screen_size[0] - (buttons_x_padding * 2 + button_width * 4)) / 3
+        
+        for i, choice in enumerate(self.question.choices):
+            choice_text = self.lg_font.render(("A", "B", "C", "D")[i] + " " + str(choice), 1, Color.BLACK)
+            self.buttons.append(Option_Button(buttons_x_padding + button_width * i + buttons_gap * i,
+                                              screen_size[1] - button_height - buttons_y_padding,
+                                              button_width, button_height, choice_text, str(choice)))
```

Comment:
I think the argument list should be pushed to a new line as that'll make it more readable.